app/components/LeftPaneEntries/TableAndVisualItem/index.tsx

```diff
@@ -0,0 +1,257 @@
+                    onClick={handleSubmit}
+                    disabled={requiredStringCondition(excerpt) !== undefined}
+                >
+                    Done
+                </Button>
+            )}
+        >
+            <ExcerptTextArea
+                className={styles.excerptTextArea}
+                name="modified-excerpt"
+                value={excerpt}
+                onChange={setExcerpt}
+                rows={10}
+            />
+        </Container>
+    );
+}
+
+interface TableAndVisualItemProps {
+    attachmentId: string;
+    onClick?: (dataId: LeadPreviewAttachmentType['id']) => void;
+    onApproveButtonClick?: QuickActionButtonProps<string>['onClick'];
+    onDiscardButtonClick?: QuickActionButtonProps<string>['onClick'];
+    className?: string;
+    disableApproveButton?: boolean;
+    disableDiscardButton?: boolean;
+    disableClick?: boolean;
+    errored?: boolean;
+    deleted?: boolean;
+    stale?: boolean;
+    data: LeadPreviewAttachmentType;
+    isActive?: boolean;
+    entryId: string | undefined;
+}
+
+function TableAndVisualItem(props: TableAndVisualItemProps) {
+    const {
+        className,
+        attachmentId,
+        onClick,
+        onApproveButtonClick,
+        onDiscardButtonClick,
+        disableApproveButton,
+        disableDiscardButton,
+        disableClick,
+        deleted,
+        errored,
+        stale,
+        data,
+        isActive,
+        entryId,
+    } = props;
+
+    const entryTypeFromLead = useMemo((): EntryTagTypeEnum | undefined => {
+        if (data?.type === 'IMAGE' || data?.type === 'XLSX') {
+            return 'IMAGE';
+        }
+        return undefined;
+    }, [data]);
+
+    const handleClick = React.useCallback(() => {
+        if (onClick && !disableClick) {
+            onClick(attachmentId);
+        }
+    }, [attachmentId, onClick, disableClick]);
+
+    if (isNotDefined(entryId)) {
+        return (
+            <div
+                className={styles.clickableArea}
+                onClick={handleClick}
+                role="presentation"
+            >
+                <Container
+                    headerActions={data?.type === 'XLSX' ? (
+                        <QuickActionLink
+                            title="Open external"
+                            to={data?.file?.url || ''}
+                        >
+                            <BsDownload />
+                        </QuickActionLink>
+                    ) : undefined}
+                >
+                    {isDefined(entryTypeFromLead) && (
+                        <ExcerptInput
+                            value={data?.type}
+                            entryType={entryTypeFromLead}
+                            image={undefined}
+                            entryAttachment={undefined}
+                            imageRaw={data?.filePreview?.url ?? ''}
+                            readOnly
+                        />
+                    )}
```

Comment:
Even after the update in way we are fetching images, ExcerptInput must get the CanonicalImageLink, we should not replace the function if imageRaw. ImageRaw is used to pass base64 coded image data after screenshot is taken. Consult with @tnagorra if necessary for proper props update in ExcerptInput.

Reply:
We can use imageRaw but maybe rename the prop.
base64 encoded images is also a uri